python/cudf/cudf/_cuda/gpu.pyx

```diff
@@ -0,0 +1,272 @@
+    cudaDevAttrPageableMemoryAccess = \
+        <c_attr> cudaDeviceAttr.cudaDevAttrPageableMemoryAccess
+    cudaDevAttrConcurrentManagedAccess = \
+        <c_attr> cudaDeviceAttr.cudaDevAttrConcurrentManagedAccess
+    cudaDevAttrComputePreemptionSupported = \
+        <c_attr> cudaDeviceAttr.cudaDevAttrComputePreemptionSupported
+    cudaDevAttrCanUseHostPointerForRegisteredMem = \
+        <c_attr> cudaDeviceAttr.cudaDevAttrCanUseHostPointerForRegisteredMem
+    cudaDevAttrReserved92 = \
+        <c_attr> cudaDeviceAttr.cudaDevAttrReserved92
+    cudaDevAttrReserved93 = \
+        <c_attr> cudaDeviceAttr.cudaDevAttrReserved93
+    cudaDevAttrReserved94 = \
+        <c_attr> cudaDeviceAttr.cudaDevAttrReserved94
+    cudaDevAttrCooperativeLaunch = \
+        <c_attr> cudaDeviceAttr.cudaDevAttrCooperativeLaunch
+    cudaDevAttrCooperativeMultiDeviceLaunch = \
+        <c_attr> cudaDeviceAttr.cudaDevAttrCooperativeMultiDeviceLaunch
+    cudaDevAttrMaxSharedMemoryPerBlockOptin = \
+        <c_attr> cudaDeviceAttr.cudaDevAttrMaxSharedMemoryPerBlockOptin
+    cudaDevAttrCanFlushRemoteWrites = \
+        <c_attr> cudaDeviceAttr.cudaDevAttrCanFlushRemoteWrites
+    cudaDevAttrHostRegisterSupported = \
+        <c_attr> cudaDeviceAttr.cudaDevAttrHostRegisterSupported
+    cudaDevAttrPageableMemoryAccessUsesHostPageTables = \
+        <c_attr> \
+        cudaDeviceAttr.cudaDevAttrPageableMemoryAccessUsesHostPageTables
+    cudaDevAttrDirectManagedMemAccessFromHost = \
+        <c_attr> cudaDeviceAttr.cudaDevAttrDirectManagedMemAccessFromHost
+
+
+def driverGetVersion():
+    """
+    Returns in the latest version of CUDA supported by the driver.
+    The version is returned as (1000 major + 10 minor). For example,
+    CUDA 9.2 would be represented by 9020. If no driver is installed,
+    then 0 is returned as the driver version.
+
+    This function returns -1 if driver version is NULL.
+    """
+    cdef int version
+    status = cudaDriverGetVersion(&version)
+    return -1 if status != 0 else version
+
+
+def runtimeGetVersion():
+    """
+    Returns the version number of the current CUDA Runtime instance.
+    The version is returned as (1000 major + 10 minor). For example,
+    CUDA 9.2 would be represented by 9020.
+
+    This function returns -1 if runtime version is NULL.
+    """
+
+    cdef int version
+    status = cudaRuntimeGetVersion(&version)
+    return -1 if status != 0 else version
+
+
+def getDeviceCount():
+    """
+    Returns the number of devices with compute capability greater or
+    equal to 2.0 that are available for execution.
+
+    This function returns -1 if NULL device pointer is assigned.
+    """
+
+    cdef int count
+    status = cudaGetDeviceCount(&count)
+    return -1 if status != 0 else count
+
+
+def getDeviceAttribute(attr, device):
+    """
+    Returns information about the device.
+
+    Parameters
+        attr
+            Device attribute to query
+        device
+            Device number to query
```

Comment:
Specify types of these Parameters and fix docstring formatting to properly denote `Parameters` section